src/react-components/ory/user-auth-card.tsx

```diff
@@ -385,17 +385,19 @@ export const UserAuthCard = ({
       $flow = LinkSection({
         nodes: flow.ui.nodes,
       })
-      message = {
-        text: intl.formatMessage({
-          id: "verification.registration-label",
-          defaultMessage: "Don't have an account?",
-        }),
-        buttonText: intl.formatMessage({
-          id: "verification.registration-button",
-          defaultMessage: "Sign up",
-        }),
-        url: additionalProps.signupURL,
-        dataTestId: "cta-link",
+      if (additionalProps.signupURL) {
```

Comment:
great stuff, could you add a story and a test for this? 

Playwright component tests are here: https://github.com/ory/elements/blob/main/src/react-components/ory/user-auth-card.spec.tsx

And Storybook tests are here:
https://github.com/ory/elements/blob/main/src/stories/Ory/Auth.stories.tsx